inst/qml/TTestIndependentSamples.qml

```diff
@@ -98,8 +98,14 @@ Form
 	{
 		title: qsTr("Assumption Checks")
 		CheckBox { name: "normalityTests";				label: qsTr("Normality")					}
-		CheckBox { name: "equalityOfVariancesTests";	label: qsTr("Equality of variances")		}
-	}
+		CheckBox { name: "equalityOfVariancesTests";	label: qsTr("Equality of variances")
+		  RadioButtonGroup {
+				name: "equalityOfVarianceType";
+				RadioButton { value: "brownForsythe"; label: qsTr("Brown-Forsythe") ; checked: true }
+				RadioButton { value: "levene"; label: qsTr("Levene's") }
+	                     }
+	           }
```

Comment:
Just a formatting comment: If the Qml component does not fit on a single line, you can indeed separate it on multiple ones. In that case, you do not need to separate properties on individual lines by `;`, and brackets `{` `}` should be on separate lines. See https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-style-example.qml for style examples.